[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for org.apache.maven.shared:maven-shared-utils:3.4.3-SNAPSHOT: The following artifacts could not be resolved: org.apache.maven.shared:maven-shared-components:pom:47 (absent): Could not transfer artifact org.apache.maven.shared:maven-shared-components:pom:47 from/to central (https://repo.maven.apache.org/maven2): PKIX path validation failed: java.security.cert.CertPathValidatorException: validity check failed and 'parent.relativePath' points at wrong local POM @ line 23, column 11
 @

--- a/src/main/java/org/apache/maven/shared/utils/xml/XmlWriterUtil.java
+++ b/src/main/java/org/apache/maven/shared/utils/xml/XmlWriterUtil.java
@@ -221,7 +221,7 @@ public class XmlWriterUtil {
                         line.append("-->").append(CRLF);
                         writer.writeMarkup(line.toString());
                     }
-                    line = new StringBuffer(indentation + "<!-- ");
+                    line = new StringBuilder(indentation + "<!-- ");
                     line.append(word).append(' ');
                 } else {
                     line.append(word).append(' ');

